client/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "client",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.15.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
```

Comment:
These should probably be dev dependencies as well, `npm install -D @testing-library/jest-dom @testing-library/react ...` since they are not actually required when running the software but are just required for developers.

Reply:
Ok sounds good